packages/mask/background/services/crypto/recipients.ts

```diff
@@ -18,6 +23,10 @@ export async function getRecipients(whoAmI: ProfileIdentifier): Promise<ProfileI
     return toProfileInformation(profiles).mustNotAwaitThisWithInATransaction
 }
 
+export async function setRecipients(value: ProfileRecord) {
+    await consistentPersonaDBWriteAccess((t) => createOrUpdateProfileDB(value, t))
+}
```

Comment:
Please do not add it here. You should add a "recipient" like a normal persona-profile link.

It should be located in `packages\mask\background\services\identity\`. (And you maybe don't need to add a new method. Please try to use existing method.)